antarest/study/web/study_data_blueprint.py

```diff
@@ -651,25 +654,26 @@ def set_optimization_form_values(uuid: str, field_values: OptimizationPreference
         path="/studies/{uuid}/config/adequacypatch/form",
         tags=[APITag.study_data],
         summary="Get adequacy patch config values for form",
-        response_model=AdequacyPatchFormFields,
         response_model_exclude_none=True,
     )
-    def get_adequacy_patch_form_values(uuid: str) -> AdequacyPatchFormFields:
+    def get_adequacy_patch_form_values(uuid: str) -> AdequacyPatchParameters:
         logger.info(msg=f"Getting adequacy patch config for study {uuid}")
         study = study_service.check_study_access(uuid, StudyPermissionType.READ)
         study_interface = study_service.get_study_interface(study)
-        return study_service.adequacy_patch_manager.get_field_values(study_interface)
+        return study_service.adequacy_patch_manager.get_adequacy_patch_parameters(study_interface)
 
     @bp.put(
         path="/studies/{uuid}/config/adequacypatch/form",
         tags=[APITag.study_data],
         summary="Set adequacy patch config with values from form",
     )
-    def set_adequacy_patch_form_values(uuid: str, field_values: AdequacyPatchFormFields) -> None:
+    def set_adequacy_patch_form_values(
```

Comment:
Something we should do in addition (probably on previous work too):
- deprecate those "/form" endpoints
- replace them with a plain "/studies/{uuid}/config/adequacypatch" endpoint which does exactly the same thing